language/move-prover/boogie-backend/src/lib.rs

```diff
@@ -37,8 +37,8 @@ const VECTOR_SMT_ARRAY_EXT_THEORY: &[u8] =
 const MULTISET_ARRAY_THEORY: &[u8] = include_bytes!("prelude/multiset-array-theory.bpl");
 const TABLE_ARRAY_THEORY: &[u8] = include_bytes!("prelude/table-array-theory.bpl");
 
-const BCS_MODULE: &str = "0x1::BCS";
-const EVENT_MODULE: &str = "0x1::Event";
+const BCS_MODULE: &str = "0x1::bcs";
```

Comment:
As you are already here perhaps add a TODO or issue that the absolute addresses need to go away :-/ That's halfway implemented but hasn't gone fully through yet.